libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -494,7 +493,6 @@ - (void)createAndStoreLogoutEvent:(NSError *)error user:(SFUserAccount*)user {
     NSMutableDictionary *attributes = [[NSMutableDictionary alloc] init];
     attributes[@"errorCode"] = [NSNumber numberWithInteger:error.code];
     attributes[@"errorDescription"] = error.localizedDescription;
-    [SFSDKEventBuilderHelper createAndStoreEvent:@"userLogout" userAccount:user className:NSStringFromClass([self class]) attributes:attributes];
```

Comment:
same comment from the closed PR https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3815#discussion_r1930999436